.ci/aws-instances-reaper.groovy

```diff
@@ -10,6 +10,7 @@ pipeline {
     PIPELINE_LOG_LEVEL = 'INFO'
     JOB_GIT_CREDENTIALS = "f6c7695a-671e-4f4f-a331-acdce44ff9ba"
     AWS_PROVISIONER_SECRET = 'secret/observability-team/ci/elastic-observability-aws-account-auth'
+    AWS_EC2_INSTANCES_REGION= 'us-east-2'
```

Comment:
Will the below work? Or the `withAWSEnv` configures the default region with the CSV file?

```suggestion
    AWS_EC2_INSTANCES_REGION = 'us-east-2'
    AWS_DEFAULT_REGION = 'us-east-2'
```

Reply:
I've not seen the step to configure the region (https://github.com/elastic/apm-pipeline-library/blob/master/vars/withAWSEnv.groovy).

And you are right, I guess simply declaring the AWS_DEFAULT_REGION variable would be enough, without having to re-export it using `withEnv`

Reply:
> I've not seen the step to configure the region

Sorry, I meant if the authetnication with the csv file somehow injects the value from the `csv` data structure, I just verified the csv data structure does not contain any default region:

```
 Import CSV credentials generated from the AWS web console. A CSV file is imported with the profile name matching the IAM user name. The CSV file must contain the following headers.

User Name

Access key ID

Secret access key
```

https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html